htdocs/ticket/class/ticket.class.php

```diff
@@ -218,6 +218,7 @@ class Ticket extends CommonObject
 	const STATUS_IN_PROGRESS = 3;
 	const STATUS_NEED_MORE_INFO = 5;
 	const STATUS_WAITING = 7;			// on hold
+	const STATUS_COMPLETED = 6;			// Completed by the user who has been assigned to the ticket. Waiting to be closed as solved.
```

Comment:
No need for this status. There is already an information in database to say action is complete (STATUS_IN_PROGRESS + progression = 100 should be enough to show status "Completed").
The progression has been added into the label of status.